remix-lib/src/execution/txRunner.js

```diff
@@ -102,43 +104,66 @@ class TxRunner {
       timestamp: timestamp,
       nonce: new BN(account.nonce++),
       gasPrice: new BN(1),
-      gasLimit: new BN(gasLimit, 10),
+      gasLimit: gasLimit,
```

Comment:
that is already a `BN`?

Reply:
checking now if it's a BN to ensure it's not an issue